src/subscription_manager/gui/registergui.py

```diff
@@ -444,6 +444,10 @@ def choose_initial_screen(self):
             # to subscriptions" is clicked in the gui)
             if self.info.get_property('skip-auto-bind'):
                 self.emit('finished')
+            # When golden ticket mode is used, then skit auto-bind too
+            if is_owner_using_golden_ticket():
+                log.debug('Skipping auto-bind, because contentAccessMode is equal to organization')
```

Comment:
Shouldn't this be: "Skipping auto-bind, because contentAccessMode is equal to org_environment"